docs/customization.rst

```diff
@@ -145,6 +145,23 @@ With this hook you can add additional test cases that will be executed in Hypoth
             Case(endpoint=context.endpoint, query={"foo": "bar"})
         )
 
+``add_case``
+~~~~~~~~~~~~~~~~~~~~~~~
```

Comment:
Could you, please,  reduce this line so it matches the length of the section name? it should be 12 symbols (`len("``add_case``"`)